openshift/machine-config-operator
# enhance MCO test library

**- What I did**

I added the OpenShift Build and Image client to the ClientSet used to drive the MCO's e2e tests. This will help form the foundation of the MCOs' layering testing. Note: This was previously accomplished by @mkenigs in the [MCBS branch](https://github.com/openshift/machine-config-operator/commit/d0b90c34f999dff39ad6fdd21cf44e3d0e1bc94f), but this work was never merged.

**- How to verify it**

While these changes should not directly impact the e2e test suites yet, one can run the e2e suites to verify this:

```console
$ make e2e-test
```

**- Description for the changelog**

Adds OpenShift Build and Image clients
